clients/src/main/java/org/apache/kafka/clients/consumer/internals/SubscriptionState.java

```diff
@@ -270,12 +262,12 @@ public boolean partitionsAutoAssigned() {
         return !this.subscription.isEmpty();
     }
 
-    public void consumed(TopicPartition tp, long offset) {
-        assignedState(tp).consumed(offset);
+    public void position(TopicPartition tp, long offset) {
```

Comment:
It's also worth saying that if we need to bring back this distinction in order to implement prefetching improvements, it's straightforward to bring it back. I'm also in favour of simplifying for now.